go.sum

```diff
@@ -1,13 +1,26 @@
+github.com/fsnotify/fsnotify v1.4.7 h1:IXs+QLmnXW2CcXuY+8Mzv/fWEsPGWxqefPtCP5CnV9I=
```

Comment:
Sorry had to run `go mod download all` otherwise Go 1.16 would accept to compile/test the projects. I'll happily revert that if you prefer.